.github/workflows/create-release.yaml

```diff
@@ -24,16 +24,10 @@ jobs:
         images: kaotoio/standalone
         tags: |
           type=schedule
-          type=ref,event=branch
-          type=ref,event=pr
-          type=semver,pattern={{version}}
-          type=semver,pattern={{major}}.{{minor}}
-          type=semver,pattern={{major}}
+          type=semver,pattern={{version}},value=${{ inputs.standalone-version }}
```

Comment:
It was using the branch/tag of this repo where this was running instead of the parameter!!!